AI Guidelines/iOS/FlowController.md

```
@@ -0,0 +1,113 @@
+  ```swift
+  override public func setup() -> UIViewController {
+      let viewModel = SettingsViewModel(flowController: self)
+      settingsViewModel = viewModel
+      let view = SettingsView(viewModel: viewModel)
+      return BaseHostingController(rootView: view, navbarTitle: L10n.settings_title)
+  }
+  ```
+
+---
+
+## 6. **Implement the `handleFlow(_:)` Method**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)